modules/installation/pages/uyuni-proxy-setup.adoc

```diff
@@ -0,0 +1,249 @@
+mkdir -m 700 /root/ssl-build
+cd /root/ssl-build
+scp root@PARENT:/root/ssl-build/RHN-ORG-PRIVATE-SSL-KEY .
+scp root@PARENT:/root/ssl-build/RHN-ORG-TRUSTED-SSL-CERT .
+scp root@PARENT:/root/ssl-build/rhn-ca-openssl.cnf .
+----
+
+
+[NOTE]
+====
+To keep the security chain intact, the {susemgrproxy} functionality requires the SSL certificate to be signed by the same CA as the {productname} Server certificate.
+Using certificates signed by different CAs for proxies and server is not supported.
+====
+
+
+
+[[uyuni-proxy-setup-confproxy]]
+== Run [command]``configure-proxy.sh``
+
+The [command]``configure-proxy.sh`` script will finalize the setup of your {susemgrproxy}.
+
+Now execute the interactive [command]``configure-proxy.sh`` script.
+Pressing kbd:[Enter] without further input will make the script use the default values provided between brackets ``[]``.
+Here is some information about the requested settings:
+
+{productname} Parent::
+A {productname} parent can be either another proxy or a {productname} Server.
+
+HTTP Proxy::
+A HTTP proxy enables your {productname} proxy to access the Web.
+This is needed if direct access to the Web is prohibited by a firewall.
+
+Proxy Version to Activate::
+Normally, the correct value (3.0, 3.1, 3.2, or 4.0) should be offered as a default.
+
+Traceback Email::
+An email address where to report problems.
+
+Use SSL::
+For safety reasons, press ``Y``.
+
+Do You Want to Import Existing Certificates?::
+Answer ``N``.
+This ensures using the new certificates that were copied previously from the {productname} server.
+
+Organization::
+The next questions are about the characteristics to use for the SSL certificate of the proxy.
+The organization might be the same organization that was used on the server, unless of course your proxy is not in the same organization as your main server.
+
+Organization Unit::
+The default value here is the proxy's hostname.
+
+City::
+Further information attached to the proxy's certificate.
+
+State::
+Further information attached to the proxy's certificate.
+
+Country Code::
+In the [guimenu]``country code`` field, enter the country code set during the {productname} installation.
+For example, if your proxy is in the US and your {productname} is in DE, enter `DE` for the proxy.
++
+
+[NOTE]
+====
+The country code must be two upper case letters.
+For a complete list of country codes, see https://www.iso.org/obp/ui/#search.
+====
+
+Cname Aliases (Separated by Space)::
+Use this if your proxy can be accessed through various DNS CNAME aliases.
+Otherwise it can be left empty.
+
+CA Password::
+Enter the password that was used for the certificate of your {productname} Server.
+
+Do You Want to Use an Existing SSH Key for Proxying SSH-Push Salt Minion?::
+Use this option if you want to reuse a SSH key that was used for SSH-Push Salt clients on the server.
+
+Create and Populate Configuration Channel rhn_proxy_config_1000010001?::
+Accept default ``Y``.
+
+SUSE Manager Username::
```

Comment:
Does the WebUI say this even for Uyuni?

Reply:
this needs checking

Reply:
> Does the WebUI say this even for Uyuni?

@juliogonzalez This is not about the Web UI.  Here we are trying to run the configure-proxy.sh script.  Is this considered to work with Uyuni?  Now trying and it prints:
```
g153:~ # configure-proxy.sh 
ERROR: This machine does not appear to be registered with SUSE Manager Server
```
All this needs serious re-testing.  We should fix or re-write this with a separate PR.

Reply:
I created https://github.com/uyuni-project/uyuni/issues/1926